sqlmodel/main.py

```diff
@@ -693,7 +693,10 @@ def get_sqlalchemy_type(field: Any) -> Any:
             scale=getattr(metadata, "decimal_places", None),
         )
     if issubclass(type_, uuid.UUID):
-        return Uuid
+        return GUID
```

Comment:
This type was not imported, is better to return sqlalchemy UUID type. That's why test are failing